pycalphad/tests/databases/Shishin_Fe-Sb-O-S_slag.dat

```diff
@@ -67,7 +67,7 @@
   2.00000      3.00000     0.000000     0.000000     0.000000
   1.65300
    3   2
- Fe2+                     Fe3+                     Sb3+
+ Fe+2                     Fe+3                     Sb+3
  O                        S
   2.00000      3.00000      3.00000

```

Comment:
@jorgepazsoldanpalma You added some new tests for databases containing `Be2` or `Al2`, which is good. The fact that you needed to change this database to be `Fe+2` when Thermochimica/FactSage can use the `Fe2+` version means that something about our implementation is wrong. 

Both the new `Be2`/`Al2` tests that were added should pass _and_ the existing tests that use the `Fe2+`-style should work (and should agree with the output of using `Fe2+`-style database in Thermochimica/FactSage)